beta-3-setup.md

```diff
@@ -1155,13 +1155,13 @@ As the `root` user, try running it with no options and you should see the note
 that a router is needed:
 
     osadm router
-    F0223 11:50:57.985423    2610 router.go:143] Router "router" does not exist
+    F0529 11:50:57.985423    2610 router.go:143] Router "router" does not exist
     (no service). Pass --create to install.
 
 So, go ahead and do what it says:
 
     osadm router --create
-    F0223 11:51:19.350154    2617 router.go:148] You must specify a .kubeconfig
+    F0529 11:51:19.350154    2617 router.go:148] You must specify a .kubeconfig
```

Comment:
no idea, as I just saw the different output and thought it might be due to the code change?